spec/tides-data-package.json

```diff
@@ -0,0 +1,2382 @@
+              "examples": [
+                "{\n  \"name\": \"my-nice-name\"\n}\n"
+              ]
+            },
+            "path": {
+              "propertyOrder": 30,
+              "title": "Path",
+              "description": "A reference to the data for this resource, as either a path as a string, or an array of paths as strings. of valid URIs.",
+              "oneOf": [
+                {
+                  "title": "Path",
+                  "description": "A fully qualified URL, or a POSIX file path.",
+                  "type": "string",
+                  "pattern": "^(?=^[^./~])(^((?!\\.{2}).)*$).*$",
+                  "examples": [
+                    "{\n  \"path\": \"file.csv\"\n}\n",
+                    "{\n  \"path\": \"http://example.com/file.csv\"\n}\n"
+                  ],
+                  "context": "Implementations need to negotiate the type of path provided, and dereference the data accordingly."
+                },
+                {
+                  "type": "array",
+                  "minItems": 1,
+                  "items": {
+                    "title": "Path",
+                    "description": "A fully qualified URL, or a POSIX file path.",
+                    "type": "string",
+                    "pattern": "^(?=^[^./~])(^((?!\\.{2}).)*$).*$",
+                    "examples": [
+                      "{\n  \"path\": \"file.csv\"\n}\n",
+                      "{\n  \"path\": \"http://example.com/file.csv\"\n}\n"
+                    ],
+                    "context": "Implementations need to negotiate the type of path provided, and dereference the data accordingly."
+                  },
+                  "examples": [
+                    "[ \"file.csv\" ]\n",
+                    "[ \"http://example.com/file.csv\" ]\n"
+                  ]
+                }
+              ],
+              "context": "The dereferenced value of each referenced data source in `path` `MUST` be commensurate with a native, dereferenced representation of the data the resource describes. For example, in a *Tabular* Data Resource, this means that the dereferenced value of `path` `MUST` be an array.",
+              "examples": [
+                "{\n  \"path\": [\n    \"file.csv\",\n    \"file2.csv\"\n  ]\n}\n",
+                "{\n  \"path\": [\n    \"http://example.com/file.csv\",\n    \"http://example.com/file2.csv\"\n  ]\n}\n",
+                "{\n  \"path\": \"http://example.com/file.csv\"\n}\n"
+              ]
+            },
+            "data": {
+              "propertyOrder": 230,
+              "title": "Data",
+              "description": "Inline data for this resource."
+            },
+            "schema": {
+              "propertyOrder": 40,
+              "title": "Table Schema",
+              "description": "A Table Schema for this resource, compliant with the [Table Schema](/tableschema/) specification.",
```

Comment:
add descriptions that it should be one of the TIDES table schemas

Reply:
actually - should this even be here? don't we want to reference the file-based schemas instead?

Reply:
Resolved in [3b3cd1b](https://github.com/TIDES-transit/TIDES/pull/124/commits/3b3cd1b2b03bc7a42c2fa831d7708ecaddeb13a9)